hathor/transaction/resources/create_tx.py

```diff
@@ -68,7 +69,14 @@ def render_POST(self, request):
         request.setHeader(b'content-type', b'application/json; charset=utf-8')
         set_cors(request, 'POST')
 
-        body_content = json_loadb(request.content.read())
+        try:
+            body_content = json_loadb(request.content.read())
+        except JSONDecodeError as exc:
+            return json_dumpb({
+                'success': False,
+                'message': str(exc),
+            })
+
```

Comment:
Wouldn't it be better to have `api_catch_exceptions` catch all `JSONDecodeError` and give a uniform message. JSON parsing isn't really application specific or endpoint specific, so I think it's odd that each endpoint has a different error message.

Reply:
Ok. I just reverted these changes.

Reply:
LGTM